### ISSUE TITLE
Corrected document - Phase 2 Robert Winter

### ISSUE THREAD

**Issue by Maroyafaied** [ONRR/onrr.gov-site#769]
@mcharg this came in by email, the accessibility check comes back with character encoding error that the owner is not able to correct and send me this info below. Can you check with Sid if there's away around this or how to handle it? Thanks

[Paygov--ROW-RUE-Rent_Form.pdf](https://github.com/ONRR/onrr.gov-site/files/6618943/Paygov--ROW-RUE-Rent_Form.pdf)
![image](https://user-images.githubusercontent.com/40175688/121241287-82ad3a00-c858-11eb-9adf-28ac8b25f456.png)

https://www.onrr.gov/ReportPay/PDFDocs/Paygov--ROW-RUE-Rent_Form.pdf

**Comment by mcharg** [comment]
Sent email to Sid.

**Comment by mcharg** [comment]
@Maroyafaied Here's the response I got about the error.  Who needs to be sent this?

The character encoding error usually is a font issue.  Try extracting the page, saving it as a Word file, then running the Inspect document and change the font for the items, reconvert to PDF, then run accessibility checker to see if issue is resolved.  Keep in mind that new errors may arise, but should be easily corrected.  Then, reinsert page into main PDF file.

**Comment by Maroyafaied** [comment]
@mcharg I'll email the owner and cc you.

**Comment by Maroyafaied** [comment]
@mcharg here is the corrected document from Candace, please send her the preview link when this is updated and also update the tracking sheet. Thanks
[csv_File_Rent_Instructions (6) (1).pdf](https://github.com/ONRR/onrr.gov-site/files/6624799/csv_File_Rent_Instructions.6.1.pdf)

**Comment by mcharg** [comment]
@merrellc This is updated in dev. Let me know if it looks good to push live.
http://2017onrr/reportpay/payments.htm

**Comment by merrellc** [comment]
Push live. Thank you!


​Candace Merrell
Minerals Revenue Specialist - Staff
Reference, Royalty and Production
Reference and Reporting Management
Office of Natural Resources Revenue
303-231-3517
***@***.***

Warning:  This message is intended only for use of the individual or entity to which it is addressed and may contain information that is privileged or confidential and exempt from disclosure under applicable law.  If the reader of this message is not the intended recipient or the employee or agent responsible for delivering this message to the intended recipient, you are hereby notified that any dissemination, distribution, or copying of this communication is strictly prohibited.  If you have received this communication in error, please notify the sender immediately by return e-mail.

________________________________
From: mcharg ***@***.***>
Sent: Wednesday, June 9, 2021 1:03 PM
To: ONRR/onrr.gov-site ***@***.***>
Cc: Merrell, Candace J ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [ONRR/onrr.gov-site] Corrected document - Phase 2 Robert Winter (#769)




 This email has been received from outside of DOI - Use caution before clicking on links, opening attachments, or responding.



@merrellc<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmerrellc&data=04%7C01%7Ccandace.merrell%40onrr.gov%7C889d4557285b456b34d208d92b795ef9%7C0693b5ba4b184d7b9341f32f400a5494%7C0%7C0%7C637588622551646545%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=Ldd%2BQ9ebAI0XaK3sB2s0LMcJxVoD3Nb6yvFhlHMQriY%3D&reserved=0> This is updated in dev. Let me know if it looks good to push live.
http://2017onrr/reportpay/payments.htm

**Comment by mcharg** [comment]
@merrellc Your doc is pushed lived.  It might take a bit before it shows up on the live site.

**Comment by mcharg** [comment]
Second doc approved via email and pushed live.
